test/Microsoft.Extensions.DependencyModel.Tests/DependencyContextBuilderTests.cs

```diff
@@ -176,6 +178,9 @@ public void FillsRuntimeLibraryProperties()
             lib.Dependencies.Should().OnlyContain(l => l.Name == "System.Collections" && l.Version == "3.3.3");
             lib.ResourceAssemblies.Should().OnlyContain(l => l.Path == "en-US/Pack.Age.resources.dll" && l.Locale == "en-US");
 
+            // When ProjectModel supports path in the lock file library, this should assert "path/TO/package".
```

Comment:
Because this test still has a dependency on an old version of the ProjectModel library.

The DependencyContextBuilder class comes from ProjectModel.  You can also tell because there are 'using' statements at the top of this file.